rcon/extended_commands.py

```diff
@@ -218,6 +218,21 @@ def do_add_vip(self, name, steam_id_64):
         with invalidates(Rcon.get_vip_ids):
             return super().do_add_vip(steam_id_64, name)
 
+
+    @ttl_cache(ttl=60)
+    def remove_all_vips(self):
```

Comment:
Just change to: `do_remove_all_vips` and you're good to got.
Only the methods starting with do_ set_ or get_ are picked up